packages/block-editor/README.md

```diff
@@ -207,6 +207,10 @@ _Related_
 
 Undocumented declaration.
 
+### BlockPopover
+
+Undocumented declaration.
```

Comment:
Adding a JSDoc to the component helps with this. Also the component already has a README, so you might be able to reuse the README content.